### PR TITLE
AQCU-1149: Add note on CORR and TSS reports about end dates

### PR DESCRIPTION
**Pull request by zmoore-usgs**

### CODE REVIEW

File: inst/correctionsataglance/correctionsataglance.Rmd [hunk 1]
```diff
@@ -36,6 +36,8 @@ cat(getLogo())
 
 ***
 
+Note: If the displayed end date is in the future, it's application is actually open-ended.
```
Comment (mhines-usgs) [inline inst/correctionsataglance/correctionsataglance.Rmd]: grammar nerd comment, I think it's should be its

File: inst/templates/timeseriessummary/body.mustache [hunk 2]
```diff
@@ -1,5 +1,11 @@
 <div id="tss-report">
 
+  <div class="api-notice">
+    Note: If the displayed end date is in the future, it's application is actually open-ended.
```
Comment (mhines-usgs) [inline inst/templates/timeseriessummary/body.mustache]: it's -> its

**Review by mhines-usgs — APPROVED** [review]
Approve with minor edit!

**Comment by zmoore-usgs** [comment]
Good catch!

**Pull request merged**